src/material-experimental/mdc-autocomplete/public-api.ts

```diff
@@ -24,4 +24,7 @@ export {
   MatAutocompleteActivatedEvent,
   MatAutocompleteDefaultOptions,
   MatAutocompleteSelectedEvent,
+  _MatAutocompleteBase,
```

Comment:
I don't think we should add these unless we have a good reason to export them. I would actually change the script to ignore anything that starts with `_` since we don't consider it a public API.

Reply:
I was thinking something similar, but then the migration isn't a straightforward import rename. It's unlikely that people would be depending on these symbols, but it can happen.

Reply:
_If_ we decide to keep these I'd at least like to move them to the bottom below a TODO to eventually remove them.